README.md

```diff
@@ -2,55 +2,49 @@
 
 # Project Name
 
-> Description the project.
+> This is The platform for africa's talking conference that is suppose to happen to 30th june, 
+so most of the contents that presents in this project are reference to the upcoming conference.
 
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- HTML, CSS, JAVASCRIPT
 
-## Live Demo (if available)
 
-[Live Demo Link](https://livedemo.com)
+## Live Demo 
 
+[Live Demo Link](https://sevenpros.github.io/Africas_talking_capstone/)
+![Desktop Version](https://user-images.githubusercontent.com/26649756/175357187-77e6f364-38f3-4e35-a8f9-0ce2b27a102d.jpg)
+![Mobile Version](https://user-images.githubusercontent.com/26649756/175357416-244b751b-4848-4030-ab62-0e3da54b11d0.jpg)
 
```

Comment:
- [ ]  So much work you've done but you've forgotten to update your readme.MD file as well. You Readme screen shoot image are not similar to your project webpage design. Please update your readme file relative to your project. 
- [ OPTIONAL] I found that you readme file does not have a summary recorded video of short summary about what you've done on your project. I recommend working on it as well so that if someone visit your project, will understand the short summary of your project.